### ISSUE TITLE
Accessibility: Allow configuring snack autohide duration for better accessibility

### ISSUE THREAD

**Issue by rart** [craftercms/craftercms#8221]
### Latest version

- [x] The issue is in the develop branch
- [ ] The issue is in the latest released version
- [ ] The issue is in an older release (specified in the ticket)

### Duplicates

- [x] I have searched the existing issues

### Is your enhancement request related to a problem? Please describe.

See https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable.html

### Describe the solution you'd like

Allow configuring auto hide duration (including disabling) for notifications which information is not accessible elsewhere.

**Comment by yacdaniel** [comment]
Hi @jvega190, please take a look at the snack that shows in the `Encryption Tool` (Global Menu & Site Config)

<img width="1774" height="476" alt="Image" src="https://github.com/user-attachments/assets/1497f5a1-b9d1-498b-91d1-379d93bc7d93" />

This snack is not honoring the autohide settings. Also, please investigate if there are other snacks like these. 

In the Account settings 

<img width="1516" height="562" alt="Image" src="https://github.com/user-attachments/assets/f6fedad1-592b-4ec7-8098-9580866b6019" />

- Update from a checkbox to a toggle button
- Improve the text to be `Enable User Interface animations.`

**Comment by yacdaniel** [comment]
Verified 

<img width="1217" height="525" alt="Image" src="https://github.com/user-attachments/assets/f4e41479-ffb4-4b38-a532-7883751a1a51" />
